content/en/blog/how-to-use-fission-functions-with-postgresql-database/index.md

```diff
@@ -0,0 +1,357 @@
+    NOTES:
+    CHART NAME: postgresql
+    CHART VERSION: 10.13.14
+    APP VERSION: 11.14.0
+
+    ** Please be patient while the chart is being deployed **
+
+    PostgreSQL can be accessed via port 5432 on the following DNS names from within your cluster:
+
+        postgresql.default.svc.cluster.local - Read/Write connection
+
+    To get the password for "postgres" run:
+
+        export POSTGRES_PASSWORD=$(kubectl get secret --namespace default postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+
+    To connect to your database run the following command:
+
+        kubectl run postgresql-client --rm --tty -i --restart='Never' --namespace default --image docker.io/bitnami/postgresql:11.14.0-debian-10-r17 --env="PGPASSWORD=$POSTGRES_PASSWORD" --command -- psql --host postgresql -U postgres -d postgres -p 5432
+
+    To connect to your database from outside the cluster execute the following commands:
+
+        kubectl port-forward --namespace default svc/postgresql 5432:5432 &
+        PGPASSWORD="$POSTGRES_PASSWORD" psql --host 127.0.0.1 -U postgres -d postgres -p 5432
+
+```
+
+#### Database and Table Setup
+
+For this application, we have created a database named `votedb` and a table named `votebank`. You can use the following `sql` query to create the table:
+
+``` sql
+    CREATE TABLE votebank (
+        id serial PRIMARY KEY,
+        voter_id VARCHAR ( 50 )  NOT NULL,
+        vote VARCHAR ( 50 ) NOT NULL
+    );
+```
+
+``` sql
+                                    Table "public.votebank"
+  Column  |         Type          | Collation | Nullable |               Default                
+----------+-----------------------+-----------+----------+--------------------------------------
+ id       | integer               |           | not null | nextval('votebank_id_seq'::regclass)
+ voter_id | character varying(50) |           | not null | 
+ vote     | character varying(50) |           | not null | 
+Indexes:
+    "votebank_pkey" PRIMARY KEY, btree (id)
+
+```
+
+### Environment Setup
+
+#### Python Environment setup
+
+Using PostgreSQL database with Fission functions require certain non-standard libraries and Python modules.
+Some of these require C extension to be present in the Python environment.
+Please refer to our [Fisson Python Environment](https://github.com/fission/environments/tree/master/python) guide and follow the steps to create a custom image.
+
+<br>
+
+In this case we need extra libraries like *postgresql-dev and libpq*.
+For this you need to update the `Dockerfile` and append these two libraries  in the `RUN` command.
+
+It should look like this: `RUN apk add --update --no-cache gcc python3-dev build-base libev-dev libffi-dev bash musl-dev postgresql-dev libpq`.
+
+After this create a docker image and push it to your Docker hub repository.
+
+Building the docker image for our custom Python environment. *(Replace the username with your actual username on Docker Hub.)*
+
+``` dockerfile
```

Comment:
This should be bash. As it's shell command.

Reply:
fixed